app/models/post.js

```diff
@@ -1,7 +1,18 @@
 import Model, { belongsTo, hasMany } from '@ember-data/model';
 
 export default class PostModel extends Model {
-  @belongsTo('role') generalRole;
-  @belongsTo('organization') organization;
-  @hasMany('agent-in-position', { inverse: 'position' }) agentsInPosition;
+  @belongsTo('role', {
+    inverse: null,
+  })
+  generalRole;
+
+  @belongsTo('organization', {
+    inverse: null,
```

Comment:
Not sure about this one, but it was already marked as null in the organization model:

https://github.com/lblod/frontend-contact-hub/pull/85/files#diff-fca73d59c3b45630be23c80f2f7b814ed4a897bbc0c7926a38077df566f17b67L12

Reply:
Yeah, I think it should be linked to positions. It would be better for later.

Reply:
ok, so I link those 2 together?

Reply:
The inverse here has to be the same as in the domain.json?

Reply:
Not sure about that. I still don't understand inverses in the linked data world :smile:

Reply:
I can't link this to positions without doing the inverse there I think.

Reply:
Are you sure these changes didn't break anything? 😬

Reply:
No :grimacing: I clicked through some screens, but it's hard to test everything.

Reply:
Anything specific I should take a look at?